core/src/test/scala/com/avsystem/commons/serialization/GenCodecUtilsTest.scala

```diff
@@ -0,0 +1,63 @@
+package com.avsystem.commons
+package serialization
+
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers
+
+class GenCodecUtilsTest extends AnyFunSuite with Matchers {
+  import GenCodecUtilsTest.*
+
```

Comment:
If we change behaviour of GenCodec, but not the behaviour of this class, we may miss the discrepancy. Maybe we should write a custom output that will make it simple to compare if the name is also equal to what gencodec actually outputs.